Source/Scene/Instanced3DModel3DTileContent.js

```diff
@@ -394,6 +387,9 @@ define([
                     AttributeCompression.octDecodeInRange(octNormalUp[0], octNormalUp[1], 65535, instanceNormalUp);
                     AttributeCompression.octDecodeInRange(octNormalRight[0], octNormalRight[1], 65535, instanceNormalRight);
                     hasCustomOrientation = true;
+                } else if (hasTileTransform) {
+                    // Don't apply any orientation when the tile has a transform
+                    Matrix3.clone(Matrix3.IDENTITY, instanceRotation);
```

Comment:
Right now the [i3dm spec](https://github.com/AnalyticalGraphicsInc/3d-tiles/tree/master/TileFormats/Instanced3DModel) says that if no custom orientation is given then it should be oriented with the wgs84 globe, but in the case of a tile transform we don't want to do this step because the transform should handle the rotation already.